packages/core-data/src/resolvers.js

```diff
@@ -66,6 +66,15 @@ export const getCurrentUser =
 export const getEntityRecord =
 	( kind, name, key = '', query ) =>
 	async ( { select, dispatch, registry, resolveSelect } ) => {
+		// For back-compat, we allow querying for static templates through
+		// wp_template.
+		if (
+			kind === 'postType' &&
+			name === 'wp_template' &&
+			typeof key === 'string'
+		) {
+			name = '_wp_static_template';
```

Comment:
Is the `_` at the beginning necessary?

Reply:
We can change it to whatever we want. I was wondering though if we should make the endpoint private by making it a random path. But the post type can stay the same.